### PR TITLE
add enabling go modules tip

### PR DESCRIPTION
**Pull request by dedifferentiator**
**What this PR does / why we need it**:

Add enabling go modules tip in local installation guide, in other case `make loki` crashes with `build flag -mod=vendor only valid when using modules`

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: docs/installation/local.md [hunk 1]
```diff
@@ -30,6 +30,11 @@ The community provides packages of Loki for openSUSE Linux. To install:
 
 ### Building
 
+Enable go modules
```
Comment (cyriltovena) [inline docs/installation/local.md]: I think this is required only for go < 1.13 ?

Comment (dedifferentiator) [reply docs/installation/local.md]: @cyriltovena yeah, my bad

Comment (cyriltovena) [reply docs/installation/local.md]: You can keep the PR but add a comment to explain this is required only for version below 1.13.

Comment (dedifferentiator) [reply docs/installation/local.md]: @cyriltovena I used `go1.12.8` when tried to install Loki, so probably yes, but as far as there's `go1.13+` in requirements I'm not sure this PR is actual, although I can reopen it if you think it's still worth to add this line to README

Comment (cyriltovena) [reply docs/installation/local.md]: All I'm saying is, I'm fine to have this if you explain when exactly it is required and when it is not.

**Pull request closed without merging**